recipes/pyviz_comms/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: pyviz_comms
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyviz_comms/pyviz_comms-{{ version }}.tar.gz
+  sha256: 8d10e2d91ef117bbf718addc0fa80ec7073cf9fd9f9cfc327916bdb33b9a9262
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+  
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyviz_comms
+
+about:
+  home: www.pyviz.org
+  summary: Bidirectional communication for PyViz
+  license: BSD 3-Clause
```

Comment:
Is there a license file we can package?

Reply:
@jakirkham  Sure! I've specified``license_file`` in 66ffd7e.